core/domain/email_manager.py

```diff
@@ -465,3 +465,42 @@ def send_suggestion_email(
                 recipient_id, feconf.SYSTEM_COMMITTER_ID,
                 feconf.EMAIL_INTENT_SUGGESTION_NOTIFICATION,
                 email_subject, email_body, feconf.NOREPLY_EMAIL_ADDRESS)
+
+
+def send_instant_feedback_message_email(
+        recipient_id, sender_id, message, exploration_title, exploration_id,
+        thread_title):
+    email_subject = 'New Oppia message'
```

Comment:
Oops, sorry! I did not actually get an email with your comment, so this is the first time I heard about it. I replied in the doc. I think it's a good idea.